packages/stryker/src/logging/LoggingClientContext.ts

```diff
@@ -0,0 +1,7 @@
+import { LogLevel } from 'stryker-api/core';
+
+
+export default interface LoggingClientContext {
+  readonly port: number;
```

Comment:
Why does this have a port?

Reply:
It is where to send the logging events to. I'll add a comment